gerbera-web/test/client/gerbera.updates.spec.js

```diff
@@ -309,6 +314,7 @@ describe('Gerbera Updates', function () {
         const currentTime = new Date().getTime();
         expect((currentTime - startTime) >= 95).toBeTruthy('Actual value was ' + (currentTime - startTime));
         expect((currentTime - startTime) <= 110).toBeTruthy('Actual value was ' + (currentTime - startTime));
+        GERBERA.Updates.clearAll();
```

Comment:
The `addUiTimer` tests were actually staging `window.setInterval()` accidentally...then the unit tests would get random invalid assertions.  Clearing all intervals and timeouts during these tests brought good stability to the test suite.